src/carbonio-files-ui-common/utils/utils.formatDateTime.test.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { formatDate } from './utils';
+
+describe('formatDate function', () => {
+	it('should format date based on system locale (en) and as 2-digits day, 2-digits month, 4-digits year by default', () => {
+		const date = new Date(2024, 4, 9);
+		const result = formatDate(date);
+		expect(result).toBe('05/09/2024');
+	});
+
+	it('should format date based on given locale', () => {
+		const date = new Date(2024, 4, 9);
+		const result = formatDate(date, 'it');
+		expect(result).toBe('09/05/2024');
+	});
+
+	it('should format date based on system local if invalid locale arg is set', () => {
+		const date = new Date(2024, 4, 9);
+		const result = formatDate(date, 'invalid');
```

Comment:
Update this

Reply:
What should be the behavior if the string is invalid, like "invalid"? I feel I should keep this test, and add another one for the pt-BR situation.

Reply:
Never mind, I've just understood that invalid is a valid keyword :exploding_head: